specs/latest/1.0/index.html

```diff
@@ -3892,9 +3892,19 @@ <h3><a name="FBO_ATTACHMENTS">Framebuffer Object Attachments</a></h3>
 framebuffer being framebuffer complete:
 
 <ul>
-<li> <code>COLOR_ATTACHMENT0</code> = <code>RGBA/UNSIGNED_BYTE</code> texture
-<li> <code>COLOR_ATTACHMENT0</code> = <code>RGBA/UNSIGNED_BYTE</code> texture + <code>DEPTH_ATTACHMENT</code> = <code>DEPTH_COMPONENT16</code> renderbuffer
-<li> <code>COLOR_ATTACHMENT0</code> = <code>RGBA/UNSIGNED_BYTE</code> texture + <code>DEPTH_STENCIL_ATTACHMENT</code> = <code>DEPTH_STENCIL</code> renderbuffer
+<li> renderbuffer or color-renderable texture
+<li> renderbuffer or color-renderable texture + <code>DEPTH_ATTACHMENT</code> = <code>DEPTH_COMPONENT16</code> renderbuffer
+<li> renderbuffer or color-renderable texture + <code>DEPTH_STENCIL_ATTACHMENT</code> = <code>DEPTH_STENCIL</code> renderbuffer
+</ul>
+
+A texture is color-renderable if it has any of the following format-type pairs:
+
+<ul>
+<li> <code>RGBA</code>, <code>UNSIGNED_BYTE</code>
+<li> <code>RGB</code>, <code>UNSIGNED_BYTE</code>
+<li> <code>RGBA</code>, <code>UNSIGNED_SHORT_4_4_4_4</code>
+<li> <code>RGBA</code>, <code>UNSIGNED_SHORT_4_4_4_4</code>
+<li> <code>RGB</code>, <code>UNSIGNED_SHORT_5_6_5</code>
```

Comment:
A couple of recollections:
* Early ES 2.0 implementations didn't guarantee renderability of these formats.
* I recall that ES 3.0 revoked renderability of some RGB formats, but looking again at the spec, RGB8 is in fact guaranteed to be color-renderable (ES 3.0.6 p.130).

What are the concerns we should be thinking of when asserting these formats are renderable in both WebGL 1.0 and 2.0?